Makefile

```diff
@@ -1,14 +1,18 @@
-.PHONY: all build clean run
+.PHONY: all build clean run network db
 
-all: build clean run
+all: build clean network db run
 
 build:
 	docker build -t collabtest/collabtest .
 
 clean:
-	-docker rm -f collabtest 2> /dev/null || true
+	-docker rm -f collabtest collabtestdb 2> /dev/null || true; docker network rm collabtest-network
 
-run:
-	docker run -it --rm -p 8080:8080 -v /var/run/docker.sock:/var/run/docker.sock --name collabtest collabtest/collabtest
+network:
+	docker network create collabtest-network
 
+run:
+	docker run -it --rm -p 8080:8080 -v /var/run/docker.sock:/var/run/docker.sock --name collabtest --network collabtest-network -e POSTGRES_PASSWORD=collabtest collabtest/collabtest
```

Comment:
should also make the postgres user an env variable just like its passed into the db image, so that we can use the same variable for both places.